common/djangoapps/third_party_auth/pipeline.py

```diff
@@ -134,9 +134,10 @@ def B(*args, **kwargs):
     AUTH_ENTRY_LOGIN: '/login',
     AUTH_ENTRY_REGISTER: '/register',
```

Comment:
@wedaly in your earlier PR, these login and register dispatch URLs were set to `/account/login/` and `/account/register/`. I think that was a mistake and changed them to `login` and `register`. Can you confirm that this is correct?